lib/http-bridge/test/integration/Test/Integration/Scenario/Transactions.hs

```diff
@@ -76,11 +76,22 @@ spec = do
             , expectFieldEqual status Pending
             ]
 
-        r' <- request @ApiWallet ctx ("GET", getWallet wb) Default payload
-        verify r'
+        ra <- request @ApiWallet ctx ("GET", getWallet wa) Default payload
+        verify ra
+            [ expectSuccess
+            , expectFieldEqual balanceTotal 999999831346
+            , expectFieldEqual balanceAvailable 0
+            ]
+
+        rb <- request @ApiWallet ctx ("GET", getWallet wb) Default payload
+        verify rb
             [ expectSuccess
             , expectEventually ctx balanceAvailable (oneMillionAda + 1)
             ]
+
+        verify ra
+            [ expectEventually ctx balanceAvailable 999999831346
```

Comment:
Hi @KtorZ
Where does this `999999831346` value come from, exactly, and what is its significance? Assuming it's supposed to be the same value as https://git.io/fj4GR, then would it be clearer to use a named constant, possibly with a comment to state its origin?

Reply:
All fixture wallets have exactly 1000M ADA on them (should be 1M though, but will fix in another PR). So, this amount is 1000M - cost of the last payment. Can make them constants :)